docker/run_local_docker.sh

```diff
@@ -67,8 +67,18 @@ fi
 echo "Stopping any previous Local Docker Containers"
 docker compose  -f docker/local-metadata/docker-compose-postgres.yml down
 docker compose -f docker/local-metadata/docker-compose.yml down
+if [ -d "/docker-volume" ]
+then
+    echo "Do you want to delete the mounted directories from host?"
+    echo "Please enter [y/N]"
+    read  input
+    if [[ $input == "Y" || $input == "y" ]]; then
+      rm -rf $PWD/docker-volume
+    fi
```

Comment:
How will you provide the terminal input when running in CI ?

Reply:
Done